spec/views/post_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Post views', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @post = @user1.posts.create(title: 'This is my first post', text: 'This is the content of my first post')
+    @user1.posts.create(title: 'This is my second post', text: 'This is the content of my second post')
+    @user2 = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo1', bio: 'Teacher from Poland')
+    @post.comments.create(text: 'This is the first comment', author_id: @user2.id)
+  end
+
+  describe 'post show page' do
```

Comment:
Good work implementing some tests for the show page! 

- [ ] Kindly add the tests missing on this file

- [ ] I can see the post's title.
- [ ] I can see who wrote the post.
- [ ] I can see the post body.
- [ ] I can see the username of each commentor.
- [ ] I can see the comment each commentor left.